src/utils/mutations/SetHasViewedIntroFlowMutation.js

```diff
@@ -0,0 +1,32 @@
+import { graphql } from 'react-relay'
+import callMutation from 'src/utils/mutations/callMutation'
+
+const mutation = graphql`
+  mutation SetHasViewedIntroFlowMutation($input: SetHasViewedIntroFlowInput!) {
+    setHasViewedIntroFlow(input: $input) {
+      user {
+        hasViewedIntroFlow
+      }
+    }
+  }
+`
+
+const SetHasViewedIntroFlowMutation = ({ enabled, userId }) =>
+  callMutation({
+    mutation,
+    variables: {
+      input: {
+        enabled,
+        userId,
+      },
+    },
+    optimisticResponse: {
+      __typename: 'Mutation',
```

Comment:
What are these `__typename` fields for? I glanced at the docs for optimisticResponse and didn't see them.